stock_picking_send_by_mail/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 - Sandra Figueroa Varela
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking by Mail",
+    "summary": "Send stock picking by email",
+    "version": "11.0.1.0.0",
```

Comment:
Can you add following key:

    "website": "https://github.com/OCA/stock-logistics-workflow",

Reply:
@bodedra done.